libraries/AP_HAL_ChibiOS/hwdef/scripts/STM32H750xx.py

```diff
@@ -76,6 +76,10 @@
 
     'DEFINES' : {
         'HAL_HAVE_HARDWARE_DOUBLE' : '1',
+        'HAL_WITH_MCU_MONITORING' : '1',
```

Comment:
These are correct for the H730 *not* the H750 which is essentially the same as the H743

Reply:
In that case, should we add a separate python script for H730?

Reply:
@andyp1per feel free to push changes to this branch. I think you have the hardware where you can test this.

Reply:
😄  @bugobliterator there is a separate python script fro H730

don't think I can push to this branch, but will try